web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -333,7 +326,10 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
 
             Portal.data.ActiveGeoNetworkRecordStore.instance().add(record);
         }
-        Ext.MsgBus.publish(PORTAL_EVENTS.VIEW_GEONETWORK_RECORD, record);
+
+        if (!multiSelect) {
+            Ext.MsgBus.publish(PORTAL_EVENTS.VIEW_GEONETWORK_RECORD, record);
```

Comment:
I'm not sure that firing a *global* `VIEW_GEONETWORK_RECORD` event is appropriate here?  Am I right in saying that the only action we currently take on such an event is to switch to step 2? Seems like it might be better to localise this event?  Maybe it's a pain to change this though.